guides/common/modules/proc_creating-a-content-view-filter-by-using-web-ui.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="creating-a-content-view-filter-by-using-web-ui"]
+= Creating a content view filter by using {ProjectWebUI}
+
+You can filter content views containing {client-content-type} content to include or exclude specific packages or errata.
+Filters are based on a combination of the _name_, _version_, and _architecture_.
+For examples of how to build a filter, see xref:common/modules/con_content-filter-examples.adoc#Content_Filter_Examples_{context}[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Lifecycle* > *Content Views*.
+. Select a content view.
+. On the *Filters* tab, click *Create filter*.
+. Enter a name.
+. From the *Content type* list, select a content type.
+. From the *Inclusion Type* list, select either *Include filter* or *Exclude filter*.
+. Optional: In the *Description* field, enter a description for the filter.
+. Click *Create filter* to create your content filter.
+. Click *Add {client-cv-filter} rule*.
+ifdef::katello[]
++
+For Deb content, click *Add DEB rule*.
```

Comment:
Something like this would make it prettier :)

```suggestion
ifndef::katello[]
. Click *Add {client-cv-filter} rule*.
endif::[]
ifdef::katello[]
. Add a filter rule:
* For {client-cv-filter} content, click *Add {client-cv-filter} rule*.
* For Deb content, click *Add DEB rule*.
```

Reply:
Good idea. I applied a slightly changed version.